content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-04-02 23:40
+author: emus
+blog_id: newsletter
+category: Newsletter
+
+---
+
+# Collaboration over the Internet! XX May 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of April 2020.
+
+We are always happy for contributors - just come along and talk to us in the [Comm-Team MUC](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
```

Comment:
We are always happy about contributors. Just join the discussion in our [Comm-Team group chat](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. The drafting process for each newsletter is [fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).

Reply:
@wurstsalat3000 thanks for all the reviews!